README.md

```diff
@@ -106,6 +106,7 @@ _Web Analytics_
 _Business Intelligence_
 
   * [Metabase](http://www.metabase.com/) - Simple Dashboarding and GUI Query tool, Nightly Emails and Slack Integration w/ PostgreSQL, MySQL, Redshift and other DBs. ([Source Code](https://github.com/metabase/metabase)) `AGPLv3` `Clojure`
+  * [Redash](http://redash.io) - connect to over 18 types of databases (SQL and "NoSQL"), query your data, visualize it and create dashboards. Everything has a URL that can be shared. Slack and HipChat integration. `BSD` `Python`
```

Comment:
There is a paid hosted option ([to support the development of the project](https://medium.com/@arikfr/8e8259ac80cb)), but the project is open source, free and you can install it on your own. I even make it easy with [ready made images for AWS, GCE and Docker](http://docs.redash.io/en/latest/setup.html) (along with a setup script for other types of installation).

I probably need to improve the website if you missed that :-(